includes/connection.php

```diff
@@ -74,6 +75,19 @@ function user_connection_error( &$errors ) {
 }
 
 /**
+ * Does the given user have a connection to Airstory?
+ *
+ * Note that this function does not check the validity of a connection, only whether or not one
+ * exists in the database.
+ *
+ * @param int $user_id The user ID to inspect.
+ * @return bool True if there's an _airstory_target for the given user, false otherwise.
+ */
+function has_connection( $user_id ) {
```

Comment:
The function is defined under the `Airstory\Connection` class, which starts up on line 14. If anyone else wanted to reference this function, they'd either have to `use` the namespace or call the function as `Airstory\Connection\has_connection()`).